### PR TITLE
Test integration

### PR DESCRIPTION
**Pull request by prantik1998**

### CODE REVIEW

File: src/views/LandingPage/OngoingTests/index.js [hunk 1]
```diff
@@ -58,10 +58,11 @@ function OngoingTests() {
 
     const remarks = '';
     const testId = '';
-    const totalSamples = 0;
+    const totalSample = 0;
     const prevalanceRate = 0;
     const selectedKit = 1;
     const selectedMachine = 1;
+    const maxpoolsize = 0;
```
Comment (Dheeraj-DJP) [inline src/views/LandingPage/OngoingTests/index.js:65]: Let's follow camelCase for naming. All other variables have been changed to that.

File: src/components/Upload/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ openFileDialog() {
 onFileAdded(evt) {
     const file = evt.target.files[0]
     axios.post(url["BASE_API_URL"]+'upload/',{
-        "test_id": 7,
+        "test_id":  JSON.parse(localStorage.getItem("user-login-info"))['user']['pk'],
```
Comment (Dheeraj-DJP) [inline src/components/Upload/index.js:36]: Can obtain this from the userInfo constant file. There might be some changes done with the way localStorage is stored, will be easier to manage.

File: src/components/Test/index.js [hunk 3]
```diff
@@ -15,23 +15,27 @@ export default class Test extends React.Component{
...
-    prevalanceRate:props.prevalanceRate,
-    selectedKit:props.selectedKit,
-    selectedMachine:props.selectedMachine,
-    showTriggerPrevalanceRate:false,
-    showTriggerSamples:false,
-    triggerPrevalanceRate:React.createRef(),
-    triggerSamples:React.createRef(),
-    labId :userInfo().labId,
-    showSpinner:false
+    totalsamples:props.totalSample,
```
Comment (Dheeraj-DJP) [inline src/components/Test/index.js:26]: camelCase for all these variables too.

File: src/components/TableLanding/index.js [hunk 4]
```diff
@@ -19,19 +19,21 @@ import testConstants from '../../constants/testConstants';
...
+    const [totalSample,setTotalSample] = useState(0)
+    const [user,setUser] = useState('')
+    const [prevalanceRate,setPrevalanceRate] = useState(0)
+    const [selectedkit,setSelectedKit] = useState('')
+    const [selectedmachine,setSelectedMachine] = useState('')
+    const [maxpoolsize,setMaxPoolSize] = useState(0)
+    const [completed,setcompleted] = useState(0)
+    const [download,setdownload] = useState(false)
+    const [remarks, setremarks] = useState('')
+    const toggletest = props.toggletest;
```
Comment (Dheeraj-DJP) [inline src/components/TableLanding/index.js:33]: This is mostly not required. Please check.

File: src/components/TableLanding/index.js [hunk 5]
```diff
@@ -19,19 +19,21 @@ import testConstants from '../../constants/testConstants';
...
-
+    const {showUpload, toggleUpload} = useUploadModal();
+    const [testId,setTestId] = useState('')
+    const [totalSample,setTotalSample] = useState(0)
+    const [user,setUser] = useState('')
+    const [prevalanceRate,setPrevalanceRate] = useState(0)
+    const [selectedkit,setSelectedKit] = useState('')
+    const [selectedmachine,setSelectedMachine] = useState('')
+    const [maxpoolsize,setMaxPoolSize] = useState(0)
+    const [completed,setcompleted] = useState(0)
```
Comment (Dheeraj-DJP) [inline src/components/TableLanding/index.js:30]: camelCase here too for a few variables

File: src/components/NavBar/User/newuser.js [hunk 6]
```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import {Container,Row,Col,InputGroup,FormControl,Overlay} from 'react-bootstrap';
+import Tooltip from 'react-bootstrap/Tooltip';
+import axios from 'axios';
+
+import './styles.scss'
+
+import userInfo from "../../../constants/userInfo"
```
Comment (Dheeraj-DJP) [inline src/components/NavBar/User/newuser.js:8]: Let's use getUserInfo here for uniformity and readability.

File: src/components/NavBar/User/newuser.js [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+
+
+export default class NewUser extends React.Component{
+
+    constructor(props)
+    {
+        super(props)
+        this.handleClose = props.handleClose
+        this.state={
+            first_name:null,
```
Comment (Dheeraj-DJP) [inline src/components/NavBar/User/newuser.js:20]: camelCase instead of separated by underline to keep everything uniform.

**Review by Dheeraj-DJP — APPROVED** [review]
Looks good. Have added few suggestions for formatting changes. 

Also, the ProductDemo_V2.mp4 file can be saved in 'assets' folder instead of 'constants' folder.

File: src/components/NavBar/User/newuser.js [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+                    </Row>
+                    <br/>
+                    <Row>
+                        <Col xs lg="3">
+                        Email
+                        </Col>
+                        <Col xs lg="5">
+                        <InputGroup className="mb-2">
+                            <FormControl
+                            aria-label="Username"
```
Comment (sunitakawane) [inline src/components/NavBar/User/newuser.js:146]: change aria-label for other form fields

File: src/components/NavBar/User/newuser.js [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+    }
+
+    addlabmember()
+    {
+        console.log(this.state.designation)
+        console.log(this.state.first_name)
+        console.log(this.state.last_name)
+        console.log(this.state.phone)
+        console.log(this.state.email)
+
```
Comment (sunitakawane) [inline src/components/NavBar/User/newuser.js:50]: remove console.logs if not needed

**Review by sunitakawane — COMMENTED** [review]
looks good to me.

**Pull request merged**